rskj-core/src/test/java/co/rsk/peg/feeperkb/FeePerKbIT.java

```diff
@@ -0,0 +1,497 @@
+        //It still has the previous voting round value
+        assertFeePerKbValue(currentFeePerKb);
+
+        // Again authorized callers vote
+
+        Coin fourthVoteFeePerKb = Coin.valueOf(124_000L);
+        // First vote for a new value
+        voteFeePerKb(fourthVoteFeePerKb, FeePerKbVoteCaller.FIRST_AUTHORIZED.getRskAddress());
+        //It still has the previous voting round value
+        assertFeePerKbValue(currentFeePerKb);
+
+        Coin fifthVoteFeePerKb = Coin.valueOf(125_000L);
+        // Second vote for a different value, different authorizer
+        voteFeePerKb(fifthVoteFeePerKb, FeePerKbVoteCaller.SECOND_AUTHORIZED.getRskAddress());
+        //It still has the previous voting round value
+        assertFeePerKbValue(currentFeePerKb);
+
+        Coin sixthVoteFeePerKb = Coin.valueOf(126_000L);
+        // Third vote for a different value, different authorizer
+        voteFeePerKb(sixthVoteFeePerKb, FeePerKbVoteCaller.THIRD_AUTHORIZED.getRskAddress());
+        //It still has the previous voting round value
+        assertFeePerKbValue(currentFeePerKb);
+
+        // Again authorized callers vote
+
+        Coin seventhVoteFeePerKb = Coin.valueOf(127_000L);
+        // First vote for a new value
+        voteFeePerKb(seventhVoteFeePerKb, FeePerKbVoteCaller.FIRST_AUTHORIZED.getRskAddress());
+        //It still has the previous voting round value
+        assertFeePerKbValue(currentFeePerKb);
+
+        Coin eighthVoteFeePerKb = Coin.valueOf(128_000L);
+        // Second vote for a different value, different authorizer
+        voteFeePerKb(eighthVoteFeePerKb, FeePerKbVoteCaller.SECOND_AUTHORIZED.getRskAddress());
+        //It still has the previous voting round value
+        assertFeePerKbValue(currentFeePerKb);
+
+        Coin ninthVoteFeePerKb = Coin.valueOf(129_000L);
+        // Third vote for a different value, different authorizer
+        voteFeePerKb(ninthVoteFeePerKb, FeePerKbVoteCaller.THIRD_AUTHORIZED.getRskAddress());
+        //It still has the previous voting round value
+        assertFeePerKbValue(currentFeePerKb);
+
+        // Again authorized callers vote
+
+        // First vote for a new value
+        voteFeePerKb(fifteenthFeePerKbVote, FeePerKbVoteCaller.FIRST_AUTHORIZED.getRskAddress());
+        //It still has the previous voting round value
+        assertFeePerKbValue(currentFeePerKb);
+
+        // Second vote for a same value than the previous one
+        voteFeePerKb(fifteenthFeePerKbVote, FeePerKbVoteCaller.SECOND_AUTHORIZED.getRskAddress());
+        currentFeePerKb = fifteenthFeePerKbVote;
+
+        //It still has the previous voting round value
+        assertFeePerKbValue(fifteenthFeePerKbVote);
+    }
+
+    @Test
+    @Order(16)
+    void voteFeePerKbChange_whenAuthorizerOneAndThreeVoteSameCurrentFeePerKbValueAndInBetweenAuthorizerTwoVotesDifferentValue_shouldSetValueVotedByOneAndThree() {
```

Comment:
isn't this test [duplicated](https://github.com/rsksmart/rskj/pull/2433/files#diff-72f83e7f5b9745bce799c9de61da92d8ffb07cd4646ef1da1758d11431208939R84)?

Reply:
Please let me know if the following method name explains better the sense of the test:

`voteFeePerKbChange_whenMajorityOfAuthorizersVoteTheSameValueIsAlreadySetAsCurrentFeePerKbValue_shouldSetValueVotedByMajority`

Reply:
Done! It was renamed in this way:

`voteFeePerKbChange_whenVotingSameCurrentFeePerKbValue_shouldSetValueVoted`

Reply:
sounds good! But i think you should remove [this](https://github.com/rsksmart/rskj/blob/f4e06f289f974c733e66c139fcabee30e240c30f/rskj-core/src/test/java/co/rsk/peg/feeperkb/FeePerKbIT.java#L468) part then